x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/pagination/pagination.spec.ts

```diff
@@ -0,0 +1,109 @@
+        // wait for table to be done loading
+        waitForTableLoad();
+        cy.get(getDraggableField('user.name'))
+          .first()
+          .invoke('text')
+          .should(text2 => {
+            expect(text1).not.to.eq(text2);
+          });
+      });
+    cy.get(getPageButtonSelector(0)).should('not.have.class', 'euiPaginationButton-isActive');
+    cy.get(getPageButtonSelector(2)).should('have.class', 'euiPaginationButton-isActive');
+  });
+
+  it('pagination keeps track of page results when tabs change', () => {
+    loginAndWaitForPage(HOSTS_PAGE_TABS.authentications);
+    waitForTableLoad();
+    cy.get(getPageButtonSelector(0)).should('have.class', 'euiPaginationButton-isActive');
+    let thirdPageResult: string;
+    cy.get(getPageButtonSelector(2)).click({ force: true });
+    // wait for table to be done loading
+    waitForTableLoad();
+
+    cy.get(getDraggableField('user.name'))
+      .first()
+      .invoke('text')
+      .then(text2 => {
+        thirdPageResult = `${text2}`;
+      });
+    cy.get(NAVIGATION_UNCOMMON_PROCESSES).click({ force: true });
+    waitForTableLoad();
+    // check uncommon processes table starts at 1
+    cy.get(getPageButtonSelector(0)).should('have.class', 'euiPaginationButton-isActive');
+
+    cy.get(NAVIGATION_AUTHENTICATIONS).click({ force: true });
+    waitForTableLoad();
+    // check authentications table picks up at 3
+    cy.get(getPageButtonSelector(2)).should('have.class', 'euiPaginationButton-isActive');
+    cy.get(getDraggableField('user.name'))
+      .first()
+      .invoke('text')
+      .should(text1 => {
+        expect(text1).to.eq(thirdPageResult);
+      });
+  });
+
+  /*
+   * We only want to comment this code/test for now because it can be nondeterministic
+   * when we figure out a way to really mock the data, we should come back to it
+   */
+  it('pagination resets results and page number to first page when refresh is clicked', () => {
+    loginAndWaitForPage(HOSTS_PAGE_TABS.authentications);
+    cy.get(NUMBERED_PAGINATION, { timeout: DEFAULT_TIMEOUT });
+    cy.get(getPageButtonSelector(0)).should('have.class', 'euiPaginationButton-isActive');
+    // let firstResult: string;
+    // cy.get(getDraggableField('user.name'))
+    //   .first()
+    //   .invoke('text')
+    //   .then(text1 => {
+    //     firstResult = `${text1}`;
+    //   });
+    cy.get(getPageButtonSelector(2)).click({ force: true });
+    waitForTableLoad();
+    cy.get(getPageButtonSelector(0)).should('not.have.class', 'euiPaginationButton-isActive');
+    cy.get(SUPER_DATE_PICKER_APPLY_BUTTON)
+      .last()
+      .click({ force: true });
+    waitForTableLoad();
+    cy.get(getPageButtonSelector(0)).should('have.class', 'euiPaginationButton-isActive');
+    // cy.get(getDraggableField('user.name'))
+    //   .first()
+    //   .invoke('text')
+    //   .should(text1 => {
+    //     expect(text1).to.eq(firstResult);
+    //   });
```

Comment:
I know this is already merged, but is the plan we are going to do a follow on PR to remove these commented out strings?